pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "tor"
+version = "3.6.0"
+description = "The leader of the show for r/TranscribersOfReddit!"
+authors = ["Grafeas Group Ltd. <devs@grafeas.org>"]
+license = "MIT"
+repository = "https://github.com/grafeasgroup/tor"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+sh = "^1.12"
+tor_core = {git = "https://github.com/grafeasgroup/tor_core"}
+pytest = {version = "^4.2",optional = true}
+pytest-cov = {version = "^2.6",optional = true}
```

Comment:
PyTest and PyTest-Cov seem like dev dependencies too. Do we use them at runtime for operating the bot?

Reply:
We don't need them at runtime, just for development. Here's how Poetry's installs break down:

* regular dependency: will install regardless.
* regular dependency, marked optional: will install if run from `poetry install`, will be absent from setup.py.
* dev dependency: will install if run from `poetry install`, has flag to disable and will be absent from setup.py.
* extras dependency: references regular dependencies and form the optional install versions.

For example, what I've got here renders out to this:

```python
install_requires = \
['sh>=1.12,<2.0', 'tor_core']

extras_require = \
{'ci': ['pytest>=4.2,<5.0',
        'pytest-cov>=2.6,<3.0',
        'codacy-coverage>=1.3,<2.0']}
```

Reply:
and actually, I also made a mistake about how that works. Lemme see about moving pytest under dev dependencies.